### PR TITLE
feat - added text utilities and lists component

### PR DESCRIPTION
**Pull request by Balakumar3018**
Added text utilities and lists component

Having issues with carbon code snippets so code might not looks relatable. (Site having some issues)

preview link - https://steady-choux-1a6359.netlify.app/docs.html#text-utilities-content
app link - https://steady-choux-1a6359.netlify.app/

### CODE REVIEW

File: Public/components/Lists/Lists.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+}
+.list-type-circle{
+    list-style-type: circle;
+}
+.stack-list{
+    list-style-type: none;
+    list-style: none;
+    background-color:var( --background-colorWhite);
+    margin: 0%;
+    border: 1px solid black;
```
Comment (teja538) [inline Public/components/Lists/Lists.css:21]: use variables for colors

**Review by teja538 — APPROVED** [review]
code looks good to merge
small suggestion : use separate class for iframe for styling so that you can reuse it and code snippets looks consistent

File: docs.html [hunk 2]
```diff
@@ -36,6 +36,8 @@ <h2 class="aside-heading">Components</h2>
                     <li class="list-item"><a href="#snackbar-content">Snackbar</a></li>
                     <li class="list-item"><a href="#modal-content">Modal</a></li>
                     <li class="list-item"><a href="#navigation-content">Navigation</a></li>
+                    <li class="list-item"><a href="#text-utilities-content">Text Utilities</a></li>
+                    <li class="list-item"><a href="#List-content">Lists</a></li>
```
Comment (sashaboi) [inline docs.html:40]: Could the aside-heading be set to position fixed so it is holding to the side>

**Review by sashaboi — APPROVED** [review]
Looks great! Good to merge

**Pull request merged**